docs/development.rst

```diff
@@ -131,102 +114,31 @@ Requirements
-    exit                                        # we are in the container, escape :)
-    sudo ln -s $HOME/miniforge/envs/mss-stable-env /opt/conda/envs/mss-stable-env # we need the origin location linked because hashbangs interpreters are with that path. (only once needed)
-    conda activate mss-stable-env               # activate env
-    cd workspace/MSS                            # go to your workspace MSS dir
-    export PYTHONPATH=`pwd`                     # add it to the PYTHONPATH
-    python mslib/msui/msui.py                   # test if the UI starts
-    pytest tests                                # run pytest
-
-
-After the image was configured you can use it like a self installed env ::
+Software environment for development
+....................................
 
-    xhost +local:docker                 # may be needed
-    conda activate mss-stable-env       # activate env
-    cd workspace/MSS                    # go to your workspace MSS dir
-    export PYTHONPATH=`pwd`             # add it to the PYTHONPATH
-    pytest tests                        # run pytest
+The dependencies necessary to get a working development environment for MSS are specified in pixi.toml and pixi.lock.
+This means you can get a shell with all required packages installed using::
 
+    pixi shell -e dev
 
+Afterwards, a call to e.g.::
 
-Manual Installing dependencies
-..............................
+    msui
 
-MSS is based on the software of the conda-forge channel located. The channel is predefined in Miniforge.
+will run the development version of msui.
 
-Create an environment and install the dependencies needed for the mss package::
+You can also use pixi's "run" subcommand to directly run a command in the development environment, like so::
 
-  $ mamba create -n mssdev
-  $ mamba activate mssdev
-  $ mamba install mss=$mss_version --only-deps
-
-Compare versions used in the meta.yaml between stable and develop branch and apply needed changes.::
-
-  $ git diff stable develop -- localbuild/meta.yaml
-
-
-Install requirements for  local testing
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-With sending a Pull Request our defined CIs do run all tests on github.
-You can do run tests own system too.
-
-For developers we provide additional packages for running tests, activate your env and run::
-
-  $ mamba install --file requirements.d/development.txt
-
-On linux install `xvfb` from your linux package manager.
-This can be used to run tests on an invisible virtual display by prepending the pytest call with `xvfb-run`, e.g.::
-
-  $ xvfb-run pytest ...
-
-We have implemented demodata as data base for testing. On first call of pytest a set of demodata becomes stored
-in a /tmp/mss* folder. If you have installed gitpython a postfix of the revision head is added.
+    pixi run -e dev msui
 
 
 Setup MSWMS server
```

Comment:
Please also change in the MSWMS and MSColab sections also the calls.
I wanted to send a patch but somehow I do something wrong, "There was an error committing your changes: File could not be edited" 
Next time we meet I need to become "aufschlaut"

https://iffmd.fz-juelich.de/IENAfBSeRhuFDO5VIN2s3Q#
Then we have all relevant here updated

Reply:
I´d not tried again today, this was the path


https://github.com/user-attachments/assets/a66b4c7f-3965-4fbc-ae28-5b585afd97c5